### PR TITLE
Collection of smaller layout improvements for new aggregation builder.

### PR DESCRIPTION
**Pull request by linuspahl**
## Description
This PR contains a collection of smaller layout improvements for the new aggregation builder layout.

This includes: 
- Properly aligning hover for help icon with visualization mode input label.
- Improving list styling for lists used in `Popover` in `HoverForHelp`.
- Enabling hyphenation for aggregation builder input labels and help texts
- Implementing `HoverForHelp` component instead of `HelpBlock`  group by interval help, to reduce section height.
- Aligning all checkboxes properly.
- Changing order of name input in metric config form, because it is optional.
- Marking metric name input as optional.

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: graylog2-web-interface/src/views/components/aggregationwizard/elementConfiguration/ElementConfigurationSection.tsx [hunk 1]
```diff
@@ -66,25 +72,29 @@ const Header = styled.div(({ theme }) => css`
     content: ' ';
     top: 50%;
     width: 100%;
-    border-bottom: 1px solid ${theme.utils.contrastingColor(theme.colors.global.contentBackground, 'AA')};
+    border-bottom: 1px solid grey;
```
Comment (kyleknighted) [inline graylog2-web-interface/src/views/components/aggregationwizard/elementConfiguration/ElementConfigurationSection.tsx:75]: Will this still render well if an Enterprise user updates their theme colors?

File: graylog2-web-interface/src/views/components/aggregationwizard/elementConfiguration/ElementConfigurationSection.tsx [hunk 2]
```diff
@@ -66,25 +72,29 @@ const Header = styled.div(({ theme }) => css`
...
     position: absolute;
   }
-`);
 
-const ElementTitle = styled.div(({ theme }) => css`
-  background-color: ${theme.colors.global.contentBackground};
+  button {
+    color: #1f1f1f;
```
Comment (kyleknighted) [inline graylog2-web-interface/src/views/components/aggregationwizard/elementConfiguration/ElementConfigurationSection.tsx:80]: Will this still render well if an Enterprise user updates their theme colors?

File: graylog2-web-interface/src/views/components/aggregationwizard/elementConfiguration/ElementConfigurationSection.tsx [hunk 3]
```diff
@@ -66,25 +72,29 @@ const Header = styled.div(({ theme }) => css`
...
+  }
+`;
+
+const ElementTitle = styled.div`
+  background: white;
```
Comment (kyleknighted) [inline graylog2-web-interface/src/views/components/aggregationwizard/elementConfiguration/ElementConfigurationSection.tsx:85]: Will this still render well if an Enterprise user updates their theme colors?

File: graylog2-web-interface/src/views/components/aggregationwizard/elementConfiguration/ElementConfigurationSection.tsx [hunk 4]
```diff
@@ -66,25 +72,29 @@ const Header = styled.div(({ theme }) => css`
...
   z-index: 1;
   padding-right: 8px;
-`);
+`;
 
-const ElementActions = styled.div(({ theme }) => css`
-  background-color: ${theme.colors.global.contentBackground};
+const ElementActions = styled.div`
+  background: white;
```
Comment (kyleknighted) [inline graylog2-web-interface/src/views/components/aggregationwizard/elementConfiguration/ElementConfigurationSection.tsx:91]: Will this still render well if an Enterprise user updates their theme colors?

**Comment by linuspahl** [comment]
@kyleknighted Thanks for the feedback, I fixed these is the past, but I've mixed some things during the last rebase.

**Comment by kmerz** [comment]
help-text-is-not-really-visible
I would suggest to show the help for visualization (which is the last section) upwards.

**Review by dennisoelkers — APPROVED** [review]
Looks good, works great!

**Pull request merged**